audbackend/core/api.py

```diff
@@ -15,6 +15,37 @@
 r"""Backend registry."""
 
 
+def available() -> typing.Dict[str, typing.List[Backend]]:
+    r"""List available backends.
+
+    Returns a dictionary with
+    registered backend names as keys
+    (see :func:`audbackend.register`)
+    and a list with backend instances as values
+    (see :func:`audbackend.create`).
+
+    Returns:
+        dictionary with backends
+
+    Examples:
```

Comment:
At the moment we always use

```suggestion
    Example:
```

In the future I would like to change it to `Examples` for all entries, but for now I would suggest to stick to what we do in other docstrings.